YesterdayApi/Core/Users/User.cs

```diff
@@ -1,21 +1,20 @@
-using System;
-using System.Collections.Generic;
-using YesterdayApi.Core.Users.Posts.Comments;
-using YesterdayApi.Core.Users.Posts.Reactions;
-using YesterdayApi.Core.Users.Profiles;
-using YesterdayApi.UserIdentity.Data.Core.Users;
-

```

Comment:
Wrong pr 🙆‍♂️

Reply:
Yeah, i know, but it was shouting at me.

Reply:
The gap was created by the previous PR

Reply:
Can you create another one and our check style everywhere else? It's sad that linting solution doesn't recognise it as an error.